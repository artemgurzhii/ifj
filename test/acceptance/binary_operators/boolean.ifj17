Scope
dim a as boolean
dim b as boolean
dim c as boolean

c = a and b
c = a or b
c = not a
End Scope
