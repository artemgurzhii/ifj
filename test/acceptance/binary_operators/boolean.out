.IFJcode17
JUMP Scope
LABEL Scope
CREATEFRAME
DEFVAR TF@temp_bool_1
DEFVAR TF@temp_bool_2
DEFVAR GF@a
DEFVAR GF@b
DEFVAR GF@c
AND GF@c GF@a GF@b
OR GF@c GF@a GF@b
NOT GF@c GF@a
LABEL END_IF_0
